pages/unsubscribe.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useState } from 'react';
+
+const Unsubscribe = () => {
```

Comment:
Since you used proper naming conventions here (TitleCase), I would also rename the file the same name for good practice :)

unsubscribe.js -> Unsubscribe.js